ARMeilleure/Decoders/OpCodeTable.cs

```diff
@@ -285,6 +285,7 @@ static OpCodeTable()
             SetA64("0>1011100<100001110010xxxxxxxxxx", InstName.Fcvtau_V,        InstEmit.Fcvtau_V,        OpCodeSimd.Create);
             SetA64("0x0011100x100001011110xxxxxxxxxx", InstName.Fcvtl_V,         InstEmit.Fcvtl_V,         OpCodeSimd.Create);
             SetA64("x00111100x110000000000xxxxxxxxxx", InstName.Fcvtms_Gp,       InstEmit.Fcvtms_Gp,       OpCodeSimdCvt.Create);
+            SetA64("0x00111000100001101110xxxxxxxxxx", InstName.Fcvtms_V,        InstEmit.Fcvtms_V,        OpCodeSimd.Create);
```

Comment:
```suggestion
            SetA64("0>0011100<100001101110xxxxxxxxxx", InstName.Fcvtms_V,        InstEmit.Fcvtms_V,        OpCodeSimd.Create);
```
The mask had the `sz` bit as 0 before, which means it wouldn't match encoding with it set to 1 (used for the .2D double variant of the instruction). Plus as per the manual:
> The encoding sz = 1, Q = 0 is reserved.

This combination is reserved, so the `>` and `<`  ensures that the 01 combination is invalid.
Please also update the tests to include hte `.2D` variant.